enhancements/update/duration-of-migration-to-multi-arch.md

```diff
@@ -0,0 +1,166 @@
+
+* Update a control plane: specify `HostedCluster.spec.release`, and
+* Update a data plane: specify `NodePool.spec.release`,
+
+where the value is [the image pull specification of an OCP release payload image](https://hypershift-docs.netlify.app/reference/api/#hypershift.openshift.io/v1beta1.Release).
+
+The MCO does not run on a hosted cluster and thus our proposal in this enhancement makes no improvement (or regression) for migration of a hosted cluster to multi-arch. Currently there is no way to discover if a hosted cluster is ready to serve a cross-arch node pool. The requests are mainly from the users of standalone clusters at the moment. When hosted cluster users desire the improvement, we can work with the HyperShift team for a solution.
+
+#### Standalone Clusters
+The proposal targets standalone clusters.
+
+#### Single-node Deployments or MicroShift
+The proposal should apply to SNO as CVO treats it the same as a standalone cluster.
+The version of MicroShift is managed by [dnf](https://docs.redhat.com/fr/documentation/red_hat_build_of_microshift/4.17/html-single/installing_with_an_rpm_package/index#microshift-install-rpm), not CVO and thus MicroShift does not hit the issue described here.
+
+### Implementation Details/Notes/Constraints
+
+The changes in this section are also reflected in [clusteroperator.md](../../dev-guide/cluster-version-operator/dev/clusteroperator.md).
+
+Each operator creates a pull request that includes:
+
+- in the `ClusterVersion` manifest:
+
+  - a new element with `name=operator-image` and `version=placeholder.url.oc.will.replace.this.org/placeholdernamespace:<operator-tag-name>` where `<operator-tag-name>` is the name of the tag for the operator in [image-references](./operators.md#how-do-i-ensure-the-right-images-get-used-by-my-manifests).
+  - OPTIONALLY a new element with `name=operand-image` and `version=placeholder.url.oc.will.replace.this.org/placeholdernamespace:<operand-tag-name>` where `<operand-tag-name>` is the name of the tag for an operator's operand in [image-references](./operators.md#how-do-i-ensure-the-right-images-get-used-by-my-manifests).
+
+
+- in the `deployment` manifest, environment variables or flags with a placeholder for an image pull specification that will be replaced when the payload is built.
+
+- in the operator's implementation, the operator takes the values of the environment variables or the flags and use them to populate the `clusteroperatror.status.versions`.
+
+A special operator: [machine-config-operator](https://github.com/openshift/machine-config-operator) (MCO).
+It is special because it is the last operator to upgrade in a cluster upgrade and it usually takes longest for MCO to upgrade. With the new populated elements in `clusteroperator.status.versions`, the ending point of migrating to multi-arch will become accurate which improves the overall UX the most.
```

Comment:
eventually will all operators have a similar change to: https://github.com/openshift/machine-config-operator/pull/4637 ? is that each team's responsibility then to implement this?